core/object/class_db.h

```diff
@@ -437,11 +437,11 @@ class ClassDB {
 
 #define GDREGISTER_CLASS(m_class)                    \
 	if (!GD_IS_DEFINED(ClassDB_Disable_##m_class)) { \
```

Comment:
I think there will be problems here. It will be checking for `ClassDB_Disable_core_bind::Directory` instead of `Directory`. If we can't find a way to make this macro work for both cases, then we could always add a new macro for classes in namespaces.

Reply:
@neikeq Seems there is another place where things don't work, as classes like `core_bind::File` and `core_bind::Mutex` and such aren't bound/exposed correctly. GDScript only sees them as GDScriptNativeClass. If you spot anything else, let me know.

Reply:
Noted this in the PR description.